scripts/data/src/org/akvo/gae/remoteapi/CheckSurveyStructure.java

```diff
@@ -0,0 +1,190 @@
+/*
+ *  Copyright (C) 2017 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.gae.remoteapi;
+
+import static org.akvo.gae.remoteapi.DataUtils.batchSaveEntities;
+import static org.akvo.gae.remoteapi.DataUtils.batchDelete;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+/*
+ * - Checks that all surveys, groups, questions and options are consistent
+ */
+public class CheckSurveyStructure implements Process {
+
+    // private static String ERR_MSG = "Unable to hide SurveyedLocale [%s], reason: %s";
+    int e1 = 0, e2 = 0, e3 = 0, e4 = 0, e5 = 0;
```

Comment:
Perhaps could be more readable if you give these variables more descriptive names